{\rtf1\ansi\ansicpg1252\cocoartf1504\cocoasubrtf760
{\fonttbl\f0\fnil\fcharset0 Menlo-Regular;\f1\fnil\fcharset0 Menlo-BoldItalic;\f2\fnil\fcharset0 Menlo-Bold;
}
{\colortbl;\red255\green255\blue255;\red34\green79\blue188;\red0\green0\blue0;}
{\*\expandedcolortbl;;\csgenericrgb\c13206\c30848\c73913;\csgenericrgb\c0\c0\c0;}
\margl1440\margr1440\vieww10800\viewh8400\viewkind0
\pard\tx560\tx1120\tx1680\tx2240\tx2800\tx3360\tx3920\tx4480\tx5040\tx5600\tx6160\tx6720\pardirnatural\partightenfactor0

\f0\fs34 \cf1 \cb2 \CocoaLigature0 sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (1045, "Access denied for user 'root'@'localhost' (using password: YES)")\
\

\f1\i\b \ul fix: \cb3 Make sure u supplied the correct password value to config.password\

\f0\i0\b0 \cb2 \ulnone \
\
sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (1045, "Access denied for user 'root'@'localhost' (using password: NO)")\

\f1\i\b \ul fix:\cb3  Make sure u supplied a value to the password seems password field was empty.\

\f0\i0\b0 \cb2 \ulnone \
\
sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (2003, "Can't connect to MySQL server on '127.0.0.1' (61)")\

\f1\i\b \ul fix:\cb3  Make sure you supplied the correct port number of MySQL on which it was running.\
\

\f0\i0\b0 \cb2 \ulnone self.port = int(port)\
ValueError: invalid literal for int() with base 10: ''\

\f2\b \ul fix: \cb3 Be sure to supply an int value to config.port\

\f1\i \

\f0\i0\b0 \cb2 \ulnone sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (1049, "Unknown database 'my1db'")\
\
fix:\cb3 Be sure the database existed in mysql, and supply the correct value to it.}